package.json

```diff
@@ -23,8 +23,8 @@
         "eslint-plugin-eslint-comments": "^3.2.0",
         "eslint-plugin-import": "^2.29.1",
         "eslint-plugin-prettier": "^5.2.1",
-        "hardhat": "^2.22.9",
-        "hardhat-deploy": "^0.12.4",
+        "hardhat": "^2.26.1",
```

Comment:
this implies dropped node18 support. But with ethereum-contracts already having dropped that, should be ok.